build-system/compile/compile-wrappers.js

```diff
@@ -58,19 +87,69 @@ exports.extension = function (name, version, latest, isModule, loadPriority) {
   // Use a numeric value instead of boolean. "m" stands for "module"
   const m = isModule ? 1 : 0;
   return (
-    `(self.AMP=self.AMP||[]).push({n:"${name}",ev:"${version}",l:${latest},` +
-    `${priority}` +
-    // The `function` is wrapped in `()` to avoid lazy parsing it, since it will
-    // be immediately executed anyway.
-    // See https://github.com/ampproject/amphtml/issues/3977
-    // TODO(wg-performance): At some point, the build pipeline began stripping
-    // out these parentheses. Is this optimization still relevant?
-    `v:"${VERSION}",m:${m},f:(function(AMP,_){\n` +
-    '<%= contents %>\n})});'
+    '{' +
```

Comment:
super nit: is it worth also using `removeWhitespace` here?

Reply:
`<%= contents %>` needs the whitespace. Replacing a different token to get that after removing whitespace seemed too roundabout.